scripts/read_from_s3.py

```diff
@@ -1,42 +1,36 @@
-
-from os.path import join
 import datetime
-import fsspec
-import xarray as xr
 import time
 
 import sys
+
 sys.path.append("../src")
-from UFS2ARCO import FV3Dataset
+from UFS2ARCO.fv3dataset import FV3Dataset
```

Comment:
Is this necessary? I hope it isn't since FV3Dataset is import in src/UFS2ARCO/__init__.py

Reply:
No, it's not.  
What will be an issue in the future [when we can pip install UFS2ARCO] is the use of the sys.path appendage which implies you have the source code AND that you are running the script from a location a step away from the src folder.